specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,126 @@
+| Analytics                 | Analytics Platform                | Collection, processing, and analysis of data to provide insights and support decision-making.                                                                                                                                              |
+|                           | Business Intelligence             | Tracking of performance and identification of trends via semantic models, dashboards, reports, and data visualizations.                                                                                                                    |
+|                           | Streaming Analytics               | Processing of real-time data streams to detect patterns, trends, and anomalies as they occur.                                                                                                                                              |
+|                           | Data Processing                   | Collection, transformation, and organization of data to make it useful for analysis. Includes tasks like data cleaning, integration, and aggregation.                                                                                      |
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                                                                 |
+|                           |                                   |                                                                                                                                                                                                                                            |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                                                        |
+|                           |                                   |                                                                                                                                                                                                                                            |
+| Compute                   | Virtual Machines                  | Deployment and running of various applications in a flexible computing environment.                                                                                                                                                        |
```

Comment:
The description seems off. Are we saying that's how we describe VMs or this is what we do with it? Needs to be more of the former and less of the latter.

Reply:
Maybe something along the lines of "Virtualized compute infrastructure that provides the ability to customize the Operating System and above"?

Reply:
I suppose this one's tough because it's so ubiquitous in cloud cost/usage, and so it has to be super general.

I'd like to avoid the use of the word "virtual" (or variants) to describe Virtual Machines.  How about:

> Isolated computing environments created by abstracting resources from physical hardware.

Reply:
Not sure how important Isolated here is either.

Reply:
Okay.  Going with:
```suggestion
| Compute                   | Virtual Machines                  | Computing environments created by abstracting resources from physical hardware.                                                                                                           |
```